translationRecorder/app/src/main/java/org/wycliffeassociates/translationrecorder/ProjectManager/activities/ActivityProjectManager.java

```diff
@@ -92,29 +93,36 @@
     private File mSourceAudioFile;
     private Project mProjectToExport;
     private ProjectDatabaseHelper db;
+    private boolean isIdenticonPlaying = false;
 
     @Override
     protected void onCreate(@Nullable Bundle savedInstanceState) {
         super.onCreate(savedInstanceState);
         setContentView(R.layout.activity_project_management);
 
+        pref = PreferenceManager.getDefaultSharedPreferences(this);
+        db = ((TranslationRecorderApp)getApplication()).getDatabase();
+
         Toolbar mToolbar = (Toolbar) findViewById(R.id.project_management_toolbar);
         setSupportActionBar(mToolbar);
         if (getSupportActionBar() != null) {
-            getSupportActionBar().setTitle("Project Management");
+            ImageView imageView = findViewById(R.id.identicon);
+            getSupportActionBar().setDisplayShowTitleEnabled(false);
+            int userId = pref.getInt(Settings.KEY_PROFILE, -1);
+            final User user = db.getUser(userId);
+            String svg = Jdenticon.Companion.toSvg(user.getHash(), 512, 0f);
+            imageView.setBackground(Sharp.loadString(svg).getDrawable());
+            imageView.setLayerType(View.LAYER_TYPE_SOFTWARE, null);
+            imageView.setOnClickListener(identiconPlayerClick(user.getAudio().toString()));
```

Comment:
maybe group this all into a function "initializeIdenticon" or something like that

Reply:
@sarabiaj done!